Rakefile

```diff
@@ -1,14 +1,14 @@
 require 'bundler/gem_tasks'
 
 namespace :gettext do
-  desc "Update pot file"
+  desc 'Update pot file'
   task :find do
-    require "hammer_cli_foreman_bootdisk/version"
-    require "hammer_cli_foreman_bootdisk/i18n"
+    require 'hammer_cli_foreman_salt/version'
+    require 'hammer_cli_foreman_salt/i18n'
     require 'gettext/tools'
 
-    domain = HammerCLIForemanBootdisk::I18n::LocaleDomain.new
-    GetText.update_pofiles(domain.domain_name, domain.translated_files, "#{domain.domain_name} #{HammerCLIForemanBootdisk.version.to_s}", :po_root => domain.locale_dir)
+    domain = HammerCLIForemanSalt::I18n::LocaleDomain.new
+    GetText.update_pofiles(domain.domain_name, domain.translated_files, "#{domain.domain_name} #{HammerCLIForemanSalt.version.to_s}", :po_root => domain.locale_dir)
   end
 end
 
```

Comment:
We should also run this rake task if we can to create the needed pot files

Maybe someday we want to translate it to German :-)